### PR TITLE
Board: Add arduino interfaces to nrf52840_pca10056

### PR DESCRIPTION
**Pull request by tanreb1**
Add arduino interfaces in dts to board nrf52840_10056

Fixes: #13708 

Signed-off-by: Grzegorz Bernat grz.bernat@gmail.com

### CODE REVIEW

File: boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts [hunk 1]
```diff
@@ -74,6 +74,16 @@
 	};
 };
 
+arduino_i2c: &i2c0 {
```
Comment (mike-scott) [inline boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts]: I think these should not be enabled here.  Shield DTS .overlay files will enable what they need.

(So, I would remove `status = "ok";` from all of these blocks.)

Comment (mike-scott) [reply boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts]: For example in K64F DTS: https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/frdm_k64f/frdm_k64f.dts#L77

**Comment by tanreb1** [comment]
@mike-scott  Thank you for your help. I commit changes.

**Comment by mike-scott** [comment]
@tanreb1 Can this be squashed down to 1 commit?

**Comment by tanreb1** [comment]
@mike-scott I squashed this commits and push it in new pull request [#14255](https://github.com/zephyrproject-rtos/zephyr/pull/14255) 

I think we can close this request.

**Comment by mike-scott** [comment]
ok

**Pull request closed without merging**